libraries/stdlib/common/src/generated/_Strings.kt

```diff
@@ -1747,7 +1747,7 @@ public fun CharSequence.minWithOrNull(comparator: Comparator<in Char>): Char? {
 /**
  * Returns `true` if the char sequence has no characters.
  * 
- * @sample samples.collections.Collections.Aggregates.none
+ * @sample samples.text.Strings.Aggregates.none
```

Comment:
Please make sure that these generated files are updated via `:tools:kotlin-stdlib-gen:run` command.
See https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/ReadMe.md#code-generation for details.